src/soilId/soilIdSlice.ts

```diff
@@ -151,6 +152,14 @@ const soilIdSlice = createSlice({
     builder.addCase(deleteProjectDepthInterval.fulfilled, (state, action) => {
       state.projectSettings[action.meta.arg.projectId] = action.payload;
     });
+
+    builder.addCase(fetchSoilDataForUser.fulfilled, state => {
+      state.loading = false;
+    });
```

Comment:
question: do we need a `rejected` case?

Reply:
Just added it! 👍

Reply:
Do we need it though? I'm not sure in which cases it is required. We have it for some actions but not others.

Reply:
I think we do need it; if fetching fails, loading should no longer be true.